.ci/checker-framework.groovy

```diff
@@ -109,27 +110,27 @@ private static int checkCheckerFrameworkReport(String profile) {
  * @param profile the checker framework profile to execute
  * @return A set of errors
  */
-private static List<List<String>> getCheckerFrameworkErrors(String profile) {
+private static List<List<String>> getCheckerFrameworkErrors(final String profile) {
     final List<String> checkerFrameworkLines = [] as ArrayList
     final String command = "mvn -e --no-transfer-progress clean compile -P${profile}"
     final Process process = getOsSpecificCmd(command).execute()
-    process.in.eachLine { line ->
+    process.in.eachLine { final line ->
         checkerFrameworkLines.add(line)
         println(line)
     }
     process.waitFor()
-    final List<List<String>> checkerFrameworkErrors = [][] as ArrayList
+    final List<List<String>> checkerFrameworkErrors = [] as ArrayList
```

Comment:
[][] as ArrayList is `ArrayList<Object>` which isn't the type we wanted, this works and doesn't give any violation, please verify this change.

Reply:
> please verify this change

How should we be verifying this?

Reply:
Can we use `new arrayList<>()` to keep it clear ?

Reply:
codenarc gives a violation on using that.

Reply:
I am ok disable this rule. 
I make code very confusing, I hope they will improve such rule in future

Reply:
Disabled and change to `new ArrayList<>()`